mapflow/functional/api/data_catalog_api.py

```diff
@@ -126,10 +126,13 @@ def add_mosaic_with_extent(self,
     def upload_image(self,
                      mosaic_id: UUID,
                      image_path: Union[Path, str],
+                     content_length: int,
```

Comment:
Did I get "send Content-Length: <image size in bytes> header in every upload image request" right?

Reply:
Yes, you should do it in every request, if it is possible.